llvm/lib/Analysis/DXILResource.cpp

```diff
@@ -534,18 +552,10 @@ namespace {
 class ResourceMapper {
   Module &M;
   LLVMContext &Context;
-  DXILResourceMap &Resources;
-
-  // In DXC, Record ID is unique per resource type. Match that.
```

Comment:
> In DXC, Record ID is unique per resource type. Match that.

This context seemed useful before, but I don't think it made it into the new version?

Reply:
Re-added it to where the IDs are calculated.